### PR TITLE
improved footer design

### PR DESCRIPTION
**Pull request by luckykumarirai**
## What is the change?
Improved footer design with social media icons. This footer is responsive.

## Related issue?
issue: #111 

## How was it tested?
I have tested in my local system.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Screenshot from 2021-02-10 09-53-31

### CODE REVIEW

**Comment by luckykumarirai** [comment]
I am a participant of JWOC and MWOC.

**Comment by Pranav016** [comment]
Good work, a few changes required.

* Change base branch of the PR to develop.
* The dark mode button seems to be asymmetric, you can align the social icons inline with the dark mode button so it seems symmetric.
* `Made by Pranav Mendiratta` can be removed now
* Reduce the size of the footer a little bit because it seems a little big w.r.t to the page.

**Comment by Pranav016** [comment]
I don't know if I can give points for both since for MWoC I'll have to add label and if I give you points for JWoC as well then I have to submit your PR link and they check that. 

I think you should choose one that you prefer more, otherwise I'll try to give.

File: app.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ if(process.env.NODE_ENV !== 'production'){
 }
 
 //Connecting to Mongo Database using ODM Mongoose-
-const URL = process.env.URL;
+const URL = 'mongodb://localhost:27017/blogDB';
```
Comment (Pranav016) [inline app.js:29]: Change this back to `process.env.URL`, you forgot to do that. `'mongodb://localhost:27017/blogDB'` this is only to run the web app locally.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Good work, a few changes required.

* Change base branch of the PR to develop. 
image

* The dark mode button seems to be asymmetric, you can align the social icons inline with the dark mode button so it seems symmetric.
* `Made by Pranav Mendiratta` can be removed now
* Reduce the size of the footer a little bit because it seems a little big w.r.t to the page.

**Pull request closed without merging**